README.md

```diff
@@ -3,36 +3,63 @@
 [![Gitter](https://badges.gitter.im/aws/aws-sdk-ruby.svg)](https://gitter.im/aws/aws-sdk-ruby?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge) [![Build Status](https://travis-ci.org/aws/aws-sdk-ruby.svg?branch=master)](https://travis-ci.org/aws/aws-sdk-ruby) [![Code Climate](https://codeclimate.com/github/aws/aws-sdk-ruby.svg)](https://codeclimate.com/github/aws/aws-sdk-ruby) [![Coverage Status](https://coveralls.io/repos/aws/aws-sdk-ruby/badge.svg?branch=master)](https://coveralls.io/r/aws/aws-sdk-ruby?branch=master)
 [![Dependency Status](https://www.versioneye.com/ruby/aws-sdk/badge.svg)](https://www.versioneye.com/ruby/aws-sdk)
 
-This is version 3 of the `aws-sdk` gem. Versions 1 and 2 can be found in
+This is version 3 of the `aws-sdk` gem. Version 2 can be found in
 these branches:
 
-* [aws-sdk-v1 branch](https://github.com/aws/aws-sdk-ruby/tree/aws-sdk-v1).
 * [aws-sdk-v2 branch](https://github.com/aws/aws-sdk-ruby/tree/aws-sdk-v2).
 
 ## Links of Interest
 
 * [Developer Guide](http://docs.aws.amazon.com/sdk-for-ruby/latest/DeveloperGuide/aws-ruby-sdk-about-ruby-sdk.html)
+* [AWS Developer Blog](https://aws.amazon.com/blogs/developer/category/ruby/)
 * [API Docs](http://docs.aws.amazon.com/sdkforruby/api/frames.html)
 * [Change Log](https://github.com/aws/aws-sdk-ruby/blob/master/CHANGELOG.md)
 * [Upgrading Notes](https://github.com/aws/aws-sdk-ruby/blob/master/UPGRADING.md)
 * [Gitter Channel](https://gitter.im/aws/aws-sdk-ruby)
 
-## NameError: uninitialized constant AWS
+## Upgrading from Version 2
 
-If you receive this error, you likely have upgraded to version 2 of the
-`aws-sdk` gem unintentionally. Version 2 uses the `Aws` namespace, not `AWS`.
-This allows version 1 and version 2 to be used in the same application.
+Version 3 modularizes the monolithic SDK into service specific gems. Aside from gem packaging differences, version 3 interfaces are backwards compatible with version 2.
 
-* [Additional Information](http://ruby.awsblog.com/post/TxFKSK2QJE6RPZ/Upcoming-Stable-Release-of-AWS-SDK-for-Ruby-Version-2)
-* [Migration Guide](https://github.com/aws/aws-sdk-ruby/blob/master/MIGRATING.md)
+1. If you depend on `aws-sdk`, you do not need to change anything.
+2. If you depend on `aws-sdk-resources`, you do not need to change anything. However in version 3 each service gem contains its own resource interfaces, thus it's highly recommended to switch your dependency to one of following options.
+3. If you depend on `aws-sdk-core`, you need to replace these with a dependency on one of the following options.
+
+### Options
+
+1. If you want to have every AWS service gems in your project, simply switch to `aws-sdk`
+
+```ruby
+aws-sdk ~> 3.0
+```
+
+2. If you want to specific several AWS service gems in your project, you need do as following:
+
+```ruby
+aws-sdk-s3 ~> 1.0
+aws-sdk-ec2 ~> 1.0
+...
+```
```

Comment:
Should show how `require` statements change as well. For example:

```ruby
require 'aws-sdk-core'
```

...would become...

```ruby
require 'aws-sdk'
```

We may also add a section about how V1 users need to now explicitly move to using the `aws-sdk-v1` gem and require line, rather than `aws-sdk` major version 1.